chore: bump friendsofphp/php-cs-fixer from 2 => 3

art-cg opened the pull request:
PhpStorm is nagging me about the outdated php-cs-fixer version.

Php 7.1.3 is still supported.
`"php": "^7.1.3 || ^8.0",`
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/composer.json

mtarld reviewed (changes requested):
Thanks for the update!

I think you must update the config filename:
```
Configuration file `.php_cs` is outdated, rename to `.php-cs-fixer.php`.  
```

art-cg commented:
You are rigth. I need to put more work on this. I saw this after i enabled my Github Actions.
Renaming is not the only task. The content of the file is outdated too.

ismail1432 commented:
Thanks @art-cg 
You can take a look at https://github.com/mtarld/api-platform-ms-bundle/pull/57 I implement changes to pass my PR https://github.com/mtarld/api-platform-ms-bundle/pull/52 😅

art-cg commented:
@ismail1432 Thanks for the info.
I changed the cache file to the default one and changed it in gitignore too.
I added a little helper entry in composer.json

Test on GitHub Actions looks good. Locally on windows it gives warnings for lineEndings.
`<error severity="warning" source="PHP-CS-Fixer.line_ending" message="Found violation(s) of type: line_ending"/>`
https://github.com/art-cg/api-platform-ms-bundle/runs/4176710611?check_suite_focus=true

mtarld commented on .php-cs-fixer.php:
```diff
@@ -14,5 +16,5 @@
-    ->setCacheFile(__DIR__.'/.php_cs.cache')
-;
+    ->setCacheFile(__DIR__.'/.php-cs-fixer.cache')
+;
```
Missing empty line here

art-cg replied:
I added the new line at the end

mtarld reviewed (approved):
Just a little review :slightly_smiling_face:

mtarld commented:
Thank you @art-cg!
